sound/soc/sof/intel/hda.c

```diff
@@ -414,6 +415,22 @@ void hda_dsp_dump_skl(struct snd_sof_dev *sdev, u32 flags)
 	}
 }
 
+/* dump the first 8 dwords representing the extended ROM status */
+static void hda_dsp_dump_ext_rom_status(struct snd_sof_dev *sdev)
+{
+	char msg[128];
+	int len = 0;
+	u32 value;
+	int i;
```

Comment:
Maybe declare 'i' and 'len' on same line?

Reply:
no, Pierre would definitely ask me to keep them on separate lines.